core/domain/user_jobs.py

```diff
@@ -140,10 +138,48 @@ def map(item):
             # If the user subscribes to this activity, he/she is automatically
             # subscribed to all feedback threads for this activity.
             if not activity_model.deleted:
-                threads = feedback_services.get_threadlist(activity_model.id)
-                for thread in threads:
-                    if thread['thread_id'] not in feedback_thread_ids_list:
-                        feedback_thread_ids_list.append(thread['thread_id'])
+                tracked_models_for_feedback.append(activity_model)
+
+        return most_recent_commits
```

Comment:
I incline to return a 2-tuple here in the form of (most_recent_commits, tracked_models_for_feedback), instead of relying on undocumented side-effects to lists that are passed in.